NineChronicles.Standalone/Controllers/GraphQLController.cs

```diff
@@ -92,6 +92,17 @@ public IActionResult SetPrivateKey([FromBody] SetPrivateKeyRequest request)
             var privateKey = new PrivateKey(ByteUtil.ParseHex(request.PrivateKeyString));
             StandaloneContext.NineChroniclesNodeService.PrivateKey = privateKey;
 
+            var agentAddress = privateKey.ToAddress();
+            // FIXME: `StandaloneContext.BlockChain` can be 'null'
+            var chainAgentState = StandaloneContext.BlockChain?.GetState(agentAddress);
+            if (chainAgentState != null)
+            {
+                var agentState = new AgentState((Bencodex.Types.Dictionary) chainAgentState);
+                StandaloneContext.NineChroniclesNodeService.ExceptionRenderer.RenderAgentAndAvatarAddresses(
+                    agentAddress,
+                    agentState.avatarAddresses.Values.ToList());
```

Comment:
Can `NullReferenceException` occur in this statement?

Reply:
No. Assign the `avatarAddresses` in the `AgentState` constructor.